Set checkbox and radio fill color to gray when disabled in core widgets example

jordanhalase opened the pull request:
Simple color change to the core widgets example. This sets the fill colors of the elements to gray if they are selected but disabled. It was hard to notice if they were disabled when they were still green.

![disabled_gray](https://github.com/user-attachments/assets/299f0700-9fab-4aa4-9076-f17859a60a5e)

alice-i-cecile commented on examples/ui/core_widgets.rs:
```diff
@@ -639,7 +639,7 @@ fn set_checkbox_or_radio_style(
 
     let mark_color: Color = match (disabled, checked) {
-        (true, true) => CHECKBOX_CHECK.with_alpha(0.5),
+        (true, true) => GRAY.into(),
```
`ELEMENT_FILL_DISABLED` gets my vote :)

jordanhalase commented:
Changed to `ELEMENT_` color names.

It looks like some of the "checkbox" nomenclature was left in for the radio button additions so I changed those to be consistent as well.

There could be more done to clean up this example but is out of the scope of this merge request to change the disabled fill color. Could be done later.
